specs/Addresses.md

```diff
@@ -50,37 +106,78 @@ if they do not explicitly name a parent locale will inherit from the "root" loca
+      of address field, translated to the current language
+    * aria-label - the text used to identify form controls for assistive
+      technologies for accessibility compliance, translated to the current language
+* field names
+    * a mapping from the English name of a region, locality, etc. to the
+      commonly used translation for that name translated to the current language.
+      For example, the region name "State" used in the United States would be
+      translated to "Staat" in Dutch. This would be used when the website and
+      input form is in Dutch, but the address that the site is asking for is
+      located in the United States.
 
 ### Reference and Associated material from discussion 2019-07-19
 
 * PayPal API standards
-    * [Address_portable.md](https://github.com/paypal/api-standards/blob/master/v1/schema/json/README_address.md) readme - describes a json object designed to work with html 5.1 autocomplete fields, Google libaddressinput validation data, and the AddressDoctor product from Informatica
-    * [address_portable.json](https://github.com/paypal/api-standards/blob/master/v1/schema/json/draft-04/address_portable.json) schema.  Incudes a detailed address object for more fine-grained addresses
-    * We also looked at PayPal's i18n reference application and the formats, field labels, and validations offered.
+    * [Address_portable.md](https://github.com/paypal/api-standards/blob/master/v1/schema/json/README_address.md)
+      readme
+        - describes a json object designed to work with html 5.1 autocomplete fields, Google libaddressinput
+          validation data, and the AddressDoctor product from Informatica
+    * [address_portable.json](https://github.com/paypal/api-standards/blob/master/v1/schema/json/draft-04/address_portable.json)
+      schema. Incudes a detailed address object for more fine-grained addresses
+  * We also looked at PayPal's i18n reference application and the formats, field labels, and validations offered.
 * Google Address implementations
-    * libaddressinput [AddressValidationMetadata](https://github.com/google/libaddressinput/wiki/AddressValidationMetadata) - describes the address validation metadata Goolge publishes and includes layouts, required fields, postal code validation patterns, field names in English, ISO 3166-2 sub-region codes where appropriate, subregion names, and latin layouts for Asian regions
+    * libaddressinput [AddressValidationMetadata](https://github.com/google/libaddressinput/wiki/AddressValidationMetadata)
+        - describes the address validation metadata Goolge publishes and includes layouts, required fields,
+          postal code validation patterns, field names in English, ISO 3166-2 sub-region codes where
+          appropriate, subregion names, and latin layouts for Asian regions
     * The Google libaddressinput data repository: https://chromium-i18n.appspot.com/ssl-address/data/
```

Comment:
I think the link might be `https://chromium-i18n.appspot.com/ssl-address`.  If I visit that link it displays `{}` only.